source/material_model/visco_plastic.cc

```diff
@@ -670,6 +749,21 @@ namespace aspect
                                    "surface: $v_{y}=\\sigma_{y}/(2{\\varepsilon}_{ii})$. "
                                    "This form of plasticity is commonly used in geodynamic models "
                                    "See, for example, Thieulot, C. (2011), PEPI 188, pp. 47-68. "
+                                   "\n"
+                                   "The user now has the option to linearly reduce the cohesion and "
+                                   "internal friction angle as a function of accumulated finite strain. "
+                                   "Finite strain may be calculated through tracers (Rene Gassmoeller) "
+                                   "or the compositional field finite strain plugin (Juliane Dannberg). "
+                                   "In either case, the user specifies compositional fields for each of "
+                                   "the symmetric strain tensor components, which must be listed before "
+                                   "any additional compositional fields and will reflect the following order "
```

Comment:
If you opt to make the changes above then you would not need to require a particular order.